components/script/dom/webglrenderingcontext.rs

```diff
@@ -216,6 +216,18 @@ pub struct WebGLRenderingContext {
 }
 
 impl WebGLRenderingContext {
+    #[cfg(target_env = "ohos")]
```

Comment:
I'd prefer to have a platform check in the webgl thread code, rather than changing the DOM implementation here.

Reply:
Sounds reasonable. Please check the latest commit, which reverts this change and moves it to the webgl thread.